### PR TITLE
Changes in maven parent to adapt  Google java codestyle

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
1. Configured fmt-maven-plugin to check Google java codestyle. 
2. Changed license to make it compatible with fmt-maven-plugin
now it's 
```
/*
 * Copyright (c) {DATE} {INITIAL COPYRIGHT OWNER} {OTHER COPYRIGHT OWNERS}.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Public License v1.0
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/legal/epl-v10.html
 *
 * Contributors:
 *    {INITIAL AUTHOR} - initial API and implementation and/or initial documentation
 */
```

Changes approved by ```Mike Milinkovich```

CQ for plugin https://dev.eclipse.org/ipzilla/show_bug.cgi?id=13960
### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/5772

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1067,6 +1066,11 @@
                     </configuration>
                 </plugin>
                 <plugin>
+                    <groupId>com.coveo</groupId>
```
Comment (garagatyi) [inline pom.xml]: Is this change related to the license formatting?

Comment (skabashnyuk) [inline pom.xml]: No this is related to Google java source code formatting.

Comment (garagatyi) [inline pom.xml]: Can you add this to PR description?

Comment (skabashnyuk) [inline pom.xml]: done

**Comment by benoitf** [comment]
The description is unclear.
It talks about fmt plugin but the configuration is changed for license-maven-plugin and there is an introduction of google fmt plugin.

**Comment by skabashnyuk** [comment]
Updated
